src/drive/web/modules/queries.js

```diff
@@ -63,12 +63,14 @@ const buildRecentQuery = () => ({
   definition: () =>
     Q('io.cozy.files')
       .where({
-        trashed: false,
         type: 'file',
-        updated_at: { $gt: null }
+        trashed: false,
```

Comment:
sur Contacts on fait également `trashed: { $exists: false }`, serait-ce pertinent ici aussi ?

Reply:
Non pour plusieurs raisons : 
- le $exists est à éviter car il fait un full scan in memory (cf https://docs.cozy.io/en/tutorials/data/advanced/#indexes-why-is-my-document-not-being-retrieved) 
- le `trashed` est un attribut toujours présent dans le io.cozy.files (https://github.com/cozy/cozy-doctypes/blob/master/docs/io.cozy.files.md#files) il n'est pas optionnel comme dans Contacts, donc on peut l'utiliser sans souci